spec/restaurant_spec.rb

```diff
@@ -0,0 +1,8 @@
+require './lib/models/restaurant'
+
+describe Restaurand do
```

Comment:
we dont realy know that we changes this, but it cold have happen during set up :-/